verilog/formatting/tree_unwrapper_test.cc

```diff
@@ -2116,6 +2122,233 @@ const TreeUnwrapperTestData kUnwrapModuleTestCases[] = {
                             L(1, {"end"})),
                           L(0, {"endmodule"})),
     },
+
+
+    {
+        "module: simple initial statement with function call",
+        "module m;initial aa(bb,cc,dd,ee);endmodule",
+        ModuleDeclaration(0,
+                          L(0, {"module", "m", ";"}),
+                          N(1,
+                            L(1, {"initial"}),
+                            N(2,
+                              L(2, {"aa", "("}),
+                              N(4,
+                                L(4, {"bb", ","}),
+                                L(4, {"cc", ","}),
+                                L(4, {"dd", ","}),
+                                L(4, {"ee", ")", ";"})))),
+                          L(0, {"endmodule"})),
+    },
+    {
+        "module: expressions and function calls inside if-statement headers",
+        "module m;"
+        "initial begin "
+        "if (aa(bb) == cc(dd)) a = b;"
+        "if (xx()) b = a;"
+        "end "
+        "endmodule",
+        ModuleDeclaration(0,
+                          L(0, {"module", "m", ";"}),
+                          N(1,
+                            L(1, {"initial", "begin"}),
+                            N(2,
+                              N(2,
+                                N(2,
+                                  L(2, {"if", "("}),
+                                  N(2,
+                                    L(2, {"aa", "("}),
+                                    L(4, {"bb", ")"})),
```

Comment:
Shouldn't `aa` be wrapped as `N(2+2`? and `bb` be wrapped in as `L(2+2+2`?  (same question in many similiar places)

When things 'fit' we won't see the difference of course, but this will be noticeable in other expanded cases.

It is permissible to adjust indentation of subtrees *after* the decision to expand or not has been made.  So just because we see level `K` in these tree representations doesn't mean it is guaranteed to be indented `K` in the final result.  What is important here is the *relative* indentation differences between parent child partitions (as if expansion were forced across all levels).